File: Famoser.SyncApi/Api/Communication/Request/SyncEntityRequest.cs
using System.Collections.Generic;
using Famoser.SyncApi.Api.Communication.Entities;

namespace Famoser.SyncApi.Api.Communication.Request
{
    public class SyncEntityRequest : CollectionEntityRequest
    {
        public List<SyncEntity> SyncEntities { get; set; } = new List<SyncEntity>();
    }
}

File: Famoser.SyncApi.Webpage/src/Models/Communication/Request/SyncEntityRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 04.11.2016
 * Time: 19:17
 */

namespace Famoser\SyncApi\Models\Communication\Request;


use Famoser\SyncApi\Models\Communication\Entities\SyncCommunicationCommunicationEntity;
use Famoser\SyncApi\Models\Communication\Request\Base\BaseRequest;

class SyncEntityRequest extends BaseRequest
{
    /* @var SyncCommunicationCommunicationEntity[] $SyncEntities */
    public $SyncEntities;
}
